sirepo/template/radia.py

```diff
@@ -357,12 +357,12 @@ def _build_cuboid(
     ):
     return _update_cuboid(
         _build_geom_obj('box', obj_name=name),
-        center or [0.0, 0.0, 0.0],
-        size or [1.0, 1.0, 1.0],
-        segments or [1, 1, 1],
+        center if center is not None else [0.0, 0.0, 0.0],
```

Comment:
Seems to me that this could be simplified. If _build_cuboid took `**kwargs` instead of a list of parameters, and then the caller always supplied keywords (not currently done), then an `pksetdefault` might do what you want.